output.c

```diff
@@ -1,6 +1,7 @@
 #include <pbkit/pbkit.h>
-#include <xboxrt/debug.h>
-#include <hal/fileio.h>
+#include <hal/debug.h>
+#include <winapi/fileapi.h>
+#include <winapi/handleapi.h>
```

Comment:
I personally just use windows.h (which is also suggested on the MSDN pages).

The winapi include folder is an implementation detail. If nxdk samples do this differently *I* would consider that a minor bug (although other developers might disagree).

I think it's possible that we'll eventually move files so it's `#include <fileapi.h>` (as it would be on Windows).. or this stops working completely (with only `#include <windows.h>` working).